native/cocos/audio/android/AudioPlayerProvider.cpp

```diff
@@ -57,6 +62,9 @@ static int getSystemAPILevel() {
     }
     sSystemApiLevel = apiLevel;
     return apiLevel;
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+    return INT32_MAX;
```

Comment:
return INT32_MAX?

Reply:
Yes, since this function is currently invalid, it returns a maximum value.

Reply:
Then please add TODO comment.